es-cluster-deployment.yml

```diff
@@ -13,7 +14,7 @@ metadata:
       }]
   labels:
     app: elasticsearch
-  name: elasticsearch
+    elasticsearch: elasticsearch-statefulset
```

Comment:
I don't see much benefit of having this compared to app=elasticsearch
If you want to get only this resource (the only sts) you'll do `oc get sts -l app=elasticsearch`

Reply:
You missed this one ;)

Reply:
darm, now fixed